frontend/assets/style/_variables.scss

```diff
@@ -0,0 +1,51 @@
+$spacer: 1rem; // same as Gulp $gap but using rem
+$spacer-xl: $spacer * 4; // 64px
+$spacer-lg: $spacer * 2; // 32px
+$spacer-md: $spacer;     // 16px
+$spacer-sm: $spacer / 2; // 8px
+$spacer-xs: $spacer / 4; // 4px
+
+// z-index constants
+// Why: https://css-tricks.com/handling-z-index/
+$zindex-mask: 60;
+$zindex-tooltip: 50;
+$zindex-modal: 40;
+$zindex-sidebar: 40;
+$zindex-header: 30;
+
+$size-1: 2rem;     // 32px
+$size-2: 1.5rem;   // 24px
+$size-3: 1.125rem; // 18px
+$size-4: 0.875rem; // 14px
+$size-5: 0.75rem;  // 12px
+
+// Alias for fonts
+$size-small: $size-5;
+$size-normal: $size-4;
+$size-medium: $size-3;
+$size-large: $size-2;
+$size-extra-large: $size-1;
+
+$radius: 3px;
+$radius-large: 5px;
+
+// Breakpoints
+// The container horizontal gap, which acts as the offset for breakpoints
+$gap: 16px;
+// 960, 1152, and 1344 have been chosen because they are divisible by both 12 and 16
+$tablet: 769px !default;
```

Comment:
Should one be added for `$phone` too?

Reply:
Before $tablet is phone. Then tablet limit is desktop etc...

Reply:
Ah I see